webservices/schemas.py

```diff
@@ -394,7 +394,9 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
         'ending_image_number': ma.fields.Str(),
         'pdf_url': ma.fields.Str(),
         'is_amended': ma.fields.Boolean(),
+        'document_description': ma.fields.Str(),
     },
```

Comment:
To follow up with our conversation, just put this line after 397:

`'form_type': ma.fields.Str(),`.

There is some things that automagically happen (or don't) with the schemas.